p2p/host/autonat/autonat.go

```diff
@@ -382,7 +382,7 @@ func (as *AmbientAutoNAT) probe(pi *peer.AddrInfo) {
 func (as *AmbientAutoNAT) getPeerToProbe() peer.ID {
 	peers := as.host.Network().Peers()
 	if len(peers) == 0 {
-		return ""
+		return peer.EmptyID
```

Comment:
Probably more idiomatic:
```suggestion
		return peer.ID{}
```